lib/diagnostic.js

```diff
@@ -19,22 +19,22 @@
 //
 //     `true` or `false`
 //
-let access = null; // replace null with your response
+let access = false;
```

Comment:
I know we didn't explicitly discuss this in the lesson, but the function would have access to `laserQuality` because they are declared in the same scope.